apps/web/public/index.html

```diff
@@ -4,7 +4,21 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
   <title><%= htmlWebpackPlugin.options.title %></title>
+  <meta name="description" content="GC Digital Talent is the new recruitment platform for digital and tech jobs in the Government of Canada. Apply now!">
+  <meta property="og:url" content="https://talent.canada.ca/en/">
```

Comment:
Unless we can control the locale, best to remove it?

```suggestion
  <meta property="og:url" content="https://talent.canada.ca/">
```